test/src/e2e_vm_tests/test_programs/name_shadowing/Forc.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+author  = "Emily Herbert <emily.herbert@fuel.sh>"
```

Comment:
I'll approve this but I think @adlerjohn has been changing these to `Fuel Labs <contact@fuel.sh>`

Reply:
Ah yes, please change author as above, and the license to `Apache-2.0` please @emilyaherbert